### PR TITLE
Guessing on import can get stuck in sql_api connection

### LINKED ISSUE #2942: Guessing on import can get stuck in sql_api connections
We've seen that today in some resque jobs stuck in sql_api connections to geocoding acount. Probably need to modify `services/sql-api/sql_api.rb`, used in `services/importer/lib/importer/content_guesser.rb`

See https://github.com/typhoeus/typhoeus#timeouts

### PR DESCRIPTION
**Pull request by Kartones**
Fixes #2942

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3102/))

**Comment by Kartones** [comment]
@rafatower CR please

I didn't wanted to mess up adding a new config parameter (that's why the constants) but the numbers have been talked with Devops

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3108/))

File: app/controllers/api/json/geocodings_controller.rb [hunk 1]
```diff
@@ -6,6 +6,8 @@ class Api::Json::GeocodingsController < Api::ApplicationController
 
   before_filter :load_table, only: [:create, :estimation_for]
 
+  GEOCODING_SQLAPI_CALLS_TIMEOUT = 45
```
Comment (rafatower) [inline app/controllers/api/json/geocodings_controller.rb]: I assume seconds, better leave a comment or something, no?

File: services/sql-api/sql_api.rb [hunk 2]
```diff
@@ -8,14 +8,19 @@ class SQLApi
     class SQLError        < StandardError; end
     class PermissionError < StandardError; end
 
-    attr_accessor :api_key, :username
-    attr_reader   :response_code, :response_body, :parsed_response
+    # seconds
+    CONNECT_TIMEOUT = 5
+    TIMEOUT = 60
```
Comment (rafatower) [inline services/sql-api/sql_api.rb]: Minor comment: I'd call them `DEFAULT_CONNECT_TIMEOUT` and `DEFAULT_TIMEOUT` and add `:connect_timeout` as another optional argument.

**Comment by rafatower** [comment]
:+1:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3113/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3123/))

**Pull request merged**